.rubocop.yml

```diff
@@ -0,0 +1,80 @@
+require:
+  - rubocop-minitest
+  - rubocop-performance
+
+AllCops:
+  NewCops: disable
+  Exclude:
+    - "bin/**/*"
+    - "Gemfile"
+    - "Gemfile.lock"
+
+Bundler/OrderedGems:
+  Enabled: false
+
+Layout/DotPosition:
+  EnforcedStyle: trailing
+
+Layout/EndOfLine:
+  EnforcedStyle: lf
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+Layout/EmptyLinesAroundAccessModifier:
+  EnforcedStyle: only_before
+
+Layout/LineLength:
+  Exclude:
+    - "test/exercises/*"
+    - "lib/analyzers/*/analyze.rb"
```

Comment:
Some of the analyzers have quite long line lengths (see [this line as an example](https://github.com/exercism/ruby-analyzer/blob/master/lib/analyzers/two_fer/analyze.rb#L12)). I quite like the current formatting, even though the lines are long. Let me know if you agree with this exclusion or would prefer to fix the line length.

Reply:
Yep - this is good too. Thanks! :)